README.md

```diff
@@ -32,9 +32,9 @@
 ### Requirements
 
 * `Django>=3.2`
-* `Markdown>=3.0`
+* `Markdown<3.4`
```

Comment:
maybe better keep both restrictions, `>=3.0` and `<3.4`?

Reply:
Still not working, one possible solution is to keep `bleach` for python <= 3.6, and use `nh3` for python >= 3.7.
In the `requirements.txt` file:

```
bleach ; python_version < '3.7'
nh3 ; python_version >= '3.7'
```

But, it also need to change the code.